app/controllers/api/v1/events_controller.rb

```diff
@@ -50,4 +39,13 @@ def device_token
   def event
     event_params.merge(owner: user)
   end
+
+  def find_event
+    if event = Event.find(params[:id])
```

Comment:
Think you could just do

```
def find_event
  Event.find(params[:id])
end
```

http://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find